test/cli-validator/tests/optionHandling.js

```diff
@@ -222,4 +222,70 @@ describe('cli tool - test option handling', function() {
       'Operations must have a non-empty `operationId`.'
     );
   });
+  /*
+  $ lint-openapi -c ./justWarnConfigOverride.json justWarn.yml
+
+errors
+
+  Message :   Schema must have a non-empty description.
+  Path    :   definitions.Category
+  Line    :   137
+
+  Message :   Schema must have a non-empty description.
+  Path    :   definitions.Tag
+  Line    :   149
+
+  Message :   Schema must have a non-empty description.
+  Path    :   definitions.Pet
+  Line    :   161
+
+warnings
+
+  Message :   Response schemas should be defined with a named ref.
+  Path    :   paths./pet/
+   */
+  it('should change output for overridden options when config file is manually specified', async function() {
+    const capturedText = [];
+
+    const unhookIntercept = intercept(function(txt) {
+      capturedText.push(stripAnsiFrom(txt));
+      return '';
+    });
+
+    const program = {};
+    program.args = ['./test/cli-validator/mockFiles/justWarn.yml'];
+    program.default_mode = true;
```

Comment:
As I mentioned before, `default_mode` should not be set to `true` when you are intending to use a configuration. This line should be removed

Reply:
removed.